.circleci/config.yml

```diff
@@ -36,23 +36,28 @@ job-references:
       - checkout
       - run: *setup_environment
       - run: *install_dependencies
+      - run:
+          name: "composer"
+          command: |
+            sudo mkdir ../../plugins
```

Comment:
What path does this resolve as, and why do you need to create it via sudo? 
You are `circleci` user at this point, and should be in the `/home/circleci/project` folder, so you're creating directories at `/` here, why?

Reply:
It needs sudo, because it creates a new folder in /home.
This comes from the master-theme composer config
https://github.com/greenpeace/planet4-master-theme/blob/develop/composer.json#L30
But I can't recall why we have introduced this installer path.

Reply:
Okay, this smells bad. Could you please test removing these lines and seeing what/if breaks?